demo/qat-init.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+# shellcheck disable=SC2154
+for dev in $(for pf in $ENABLING_PF_PCI_IDs; do lspci | grep "$pf" | cut -f 1 -d ' '; done); do
+  DEVPATH="/sys/bus/pci/devices/0000:$dev"
+  NUMVFS="$DEVPATH/sriov_numvfs"
+  if test -w "$NUMVFS" -a "$(cat "$NUMVFS")" -eq 0 ; then
+    tee "$NUMVFS" < "$DEVPATH/sriov_totalvfs"
+    ls -la "$NUMVFS"
```

Comment:
debug?

Reply:
What is wrong here? Sorry, I didn't get your point.!

Reply:
is `ls -la` a debug print?

Reply:
Oops. Thank you.! After changing "echo ~~ | tee ~~" to "tee ~~ < ~~"  (because shellcheck doesn't like it), I put it since I could not see any result about what's going on. Let me remove. Thanks.